tests/pr_test.go

```diff
@@ -88,11 +87,11 @@ func TestRunAdvancedExampleUpgrade(t *testing.T) {
 	randomPass := "A1" + base64.URLEncoding.EncodeToString(randomBytes)[:13]
 
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:            t,
-		TerraformDir:       "examples/advanced",
-		Prefix:             "redis-advanced-upg",
-		ResourceGroup:      resourceGroup,
-		BestRegionYAMLPath: regionSelectionPath,
+		Testing:       t,
+		TerraformDir:  "examples/advanced",
+		Prefix:        "redis-advanced-upg",
+		ResourceGroup: resourceGroup,
+		Region:        "eu-de",
```

Comment:
Why remove the region randomisation during testing?

Reply:
reverted this back, for some reason incorrect region were being selected causing test to fail
https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/pull/376#issuecomment-2239020843